cluster-install/install.sh

```diff
@@ -1005,27 +1025,31 @@ check_if_image_can_be_pulled_via_curl() {
     if [ -z "$image_repo" ]; then fatal "no image_repo given"; fi
     if [ -z "$image_tag" ]; then fatal "no image_tag given"; fi
 
-    local -a args=('-s' '-w' "\n%{http_code}")
+    local -a args=('-sS' '-w' "\n%{http_code}")
     if [ -n "$encoded_creds" ]; then
         args+=("-H" "Authorization: Basic $encoded_creds")
     fi
+    args+=("-H" "Accept: application/vnd.docker.distribution.manifest.v2+json")
 
-    local -r result="$(curl -X GET "${args[@]}" "https://$registry/v2/$image_repo/tags/list")"
+    local -r result="$(curl -X GET "${args[@]}" "https://$registry/v2/$image_repo/manifests/$image_tag" 2> "$__ERR_FILE")"
```

Comment:
This is the new way of checking if images exist; we directly fetch the manifest, which is much faster than the previous method of listing tags (especially when there's a lot of tags) when the manifest exists -- something like going from a full minute difference for some images on `docker.repo.eng.netapp.com` to practically instant.

Unfortunately, if the manifest is missing, it takes almost as long as it did with the previous method, for some reason. Still much better though since, in most cases, the manifests _will_ exist.